packages/legend-extension-dsl-serializer/style/index.scss

```diff
@@ -0,0 +1,278 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'mixins' as *;
+
+:root {
+  --color-schema-set: var(--color-blue-50);
+  --color-binding: var(--color-blue-50);
+}
+
+.icon--schema-set {
+  color: var(--color-schema-set);
+}
+
+.icon--binding {
+  color: var(--color-binding);
+}
+
+.format-editor {
```

Comment:
if we have a lot like this, probably it's better to move these to another file and import back into this main one